src/plugins/telemetryTable/ViewActions.js

```diff
@@ -51,52 +53,56 @@ let unmarkAllRows = {
     showInStatusBar: true,
     group: 'view'
 };
-let pause = {
+
+const pause = {
     name: 'Pause',
     key: 'pause-data',
     description: 'Pause real-time data flow',
     cssClass: 'icon-pause',
-    invoke: (objectPath, viewProvider) => {
-        viewProvider.getViewContext().togglePauseByButton();
+    invoke: (objectPath, view) => {
```

Comment:
Thank you for renaming these from `view` to `viewProvider`. I think there was a misunderstanding in the initial implementation, but it would certainly have caused confusion to developers.